tajo-catalog/tajo-catalog-client/src/main/java/org/apache/tajo/catalog/AbstractCatalogClient.java

```diff
@@ -598,18 +604,15 @@ public final boolean existsTable(final String tableName) {
   }
 
   @Override
-  public final boolean createIndex(final IndexDesc index) {
+  public final void createIndex(final IndexDesc index) throws DuplicateIndexException {
```

Comment:
If index creation is based on table, it may need ``Undefined(Table|Database)Exception``.